crates/bevy_app/src/main_schedule.rs

```diff
@@ -0,0 +1,127 @@
+use crate::App;
+use bevy_ecs::{
+    schedule::ScheduleLabel,
+    system::{Local, Resource},
+    world::{Mut, World},
+};
+
+/// The schedule that contains the app logic that is evaluated each tick of [`App::update()`].
+#[derive(ScheduleLabel, Clone, Debug, PartialEq, Eq, Hash)]
+pub struct Main;
+
+/// The schedule that runs before [`Startup`].
+#[derive(ScheduleLabel, Clone, Debug, PartialEq, Eq, Hash)]
+pub struct PreStartup;
+
+/// The schedule that runs once when the app starts.
+#[derive(ScheduleLabel, Clone, Debug, PartialEq, Eq, Hash)]
+pub struct Startup;
+
+/// The schedule that runs once after [`Startup`].
+#[derive(ScheduleLabel, Clone, Debug, PartialEq, Eq, Hash)]
+pub struct PostStartup;
+
+/// The schedule that runs once when the app starts.
+#[derive(ScheduleLabel, Clone, Debug, PartialEq, Eq, Hash)]
+pub struct First;
+
+/// The schedule that contains work required to make [`Update`] logic.
```

Comment:
This description is unclear and seems to be a sentence fragment.

Reply:
Examples would help a lot in this docs.

Reply:
Good calls!